### PR TITLE
Update config-sync pkg to 1.18.3

### PR DESCRIPTION
**Pull request by efiacor**
https://github.com/nephio-project/nephio/issues/800

The addition of the git.period value in the RootSync resource was added to assess a better value for this.
https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/reference/rootsync-reposync-fields#configuring-git-repo

The default value is unchanged at 15 sec.

### CODE REVIEW

File: nephio/optional/rootsync/rootsync.yaml [hunk 1]
```diff
@@ -11,3 +11,4 @@ spec:
     auth: token
     secretRef:
       name: example-cluster-name-access-token-configsync
+    period: 15s
```
Comment (arora-sagar) [inline nephio/optional/rootsync/rootsync.yaml:14]: @efiacor can you add in the PR the description of this parameter period? This way we can keep track of having this value as 15.

Comment (efiacor) [reply nephio/optional/rootsync/rootsync.yaml:14]: done

File: nephio/core/configsync/configsync.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: configmanagement.gke.io/v1
 kind: ConfigManagement
-metadata: # kpt-merge: /config-management
```
Comment (vjayaramrh) [inline nephio/core/configsync/configsync.yaml:3]: what is the reason for removing the text after `#`?

Comment (efiacor) [reply nephio/core/configsync/configsync.yaml:3]: I'm pretty sure we don't need it.
https://kpt.dev/reference/cli/pkg/update/?id=resource-identity
I'm guessing the orig pkg was created using kpt but as it stands we don't need to customize the pkg further.
@johnbelamaric @henderiw could you confirm?

**Review by liamfallon — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**